toucan_connectors/mysql/mysql_connector.py

```diff
@@ -237,17 +237,20 @@ def _connect(
                 NamedTemporaryFile(prefix='ssl_key') as ssl_key,
             ):
                 # Writing cert contents to temporary files. They're opened in wb+ mode
-                ssl_ca.write(ssl_params['ssl_ca'].encode())
-                ssl_ca.seek(0)
-                ssl_cert.write(ssl_params['ssl_cert'].encode())
-                ssl_cert.seek(0)
-                ssl_key.write(ssl_params['ssl_key'].encode())
-                ssl_key.seek(0)
+                if 'ssl_ca' in ssl_params:
+                    ssl_ca.write(ssl_params['ssl_ca'].encode())
+                    ssl_ca.seek(0)
+                if 'ssl_cert' in ssl_params:
+                    ssl_cert.write(ssl_params['ssl_cert'].encode())
+                    ssl_cert.seek(0)
+                if 'ssl_key' in ssl_params:
+                    ssl_key.write(ssl_params['ssl_key'].encode())
+                    ssl_key.seek(0)
 
                 connection_params |= {
```

Comment:
I wonder wether `ssl_verify_cert` shouldn't be set to `True` as well if we  have no client key/cert